pyomo/contrib/fme/fourier_motzkin_elimination.py

```diff
@@ -753,13 +750,13 @@ def post_process_fme_constraints(
                 obj_val = value(obj)
             # if we couldn't make it infeasible, it's useless
             if obj_val >= tolerance:
-                m.del_component(projected_constraints[i])
                 del projected_constraints[i]
             else:
                 projected_constraints[i].activate()
 
         # clean up
         m.del_component(obj)
+        del obj
```

Comment:
This is unnecessary (it implicitly happens when you reassign `obj` in the for loop in the next line)